app/scripts/modules/core/cluster/allClustersGroupings.component.html

```diff
@@ -1,8 +1,4 @@
-<div data-scroll-id="clusters-content"
-     waypoint-container
```

Comment:
It's not, it's gone, it was useless, bye bye. It's only used in the v2modalwizard now.